setup.py

```diff
@@ -2,12 +2,15 @@
 
 from distutils.extension import Extension
 from Cython.Distutils import build_ext
+# to get the proper numpy header locations (if using anaconda)
+import numpy as np
```

Comment:
So if you do `python setup.py install` before Numpy is installed, I'm worried this will cause problems. Normally, since Numpy is one of the requirements listed below, it would be installed when this file is run, not necessarily before.